src/cli/commands/init.js

```diff
@@ -96,7 +96,7 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
     {
       key: 'private',
       question: 'private',
-      default: '',
+      default: config.getOption('init-private') || '',
```

Comment:
any reason this shouldn't default to `false`?

Reply:
No idea :D

Reply:
Hm, because, default this's not set yet. It will not add to `package.json`. If set it's `false`, always it will be add to `package.json`: `private: false`.